packages/p2p/src/pages/my-profile/payment-methods/payment-methods-list/payment-methods-list.jsx

```diff
@@ -137,48 +129,6 @@ const PaymentMethodsList = () => {
                     </div>
                 </MobileFullPageModal>
             </MobileWrapper>
-            <Modal
```

Comment:
Nice, why this modal is here 😆

Reply:
yeah there are some modals that havent been moved out yet 👀